### PR TITLE
Reorder and fix typos in cas.x

### PR DESCRIPTION
**Pull request by sebras**
Pages 16 and 17 were probably scanned out of order, hence the contents of `cas.x` were broken. With these set of commits I think the situation improves.

Let me know what you think.

### CODE REVIEW

File: scans/README.md [hunk 1]
```diff
@@ -29,10 +29,8 @@ Details of the files:
 * maksys.s: page 64  of 01-s1.pdf
 * trysys.s: page 65  of 01-s1.pdf
 * scope.v: pages 3-18 of 03-scope.pdf
-* cas.s: pages 1-17 of 04-cas.pdf
-* adm.s: pages 2-11  of 05-1-4.pdf
-* ald.s: pages 13-17 of 05-1-4.pdf
-* apr.s: pages 19-30 of 05-1-4.pdf
```
Comment (philbudne) [inline scans/README.md]: did you mean for these lines to go away?

Comment (sebras) [reply scans/README.md]: Hold on. I need to check. :)

Comment (sebras) [reply scans/README.md]: Nope I did not.

Comment (sebras) [reply scans/README.md]: I resolved this in my new push. Thanks for looking thoroughly. :)

**Pull request merged**